src/Library/demos/Drag and Drop/main.js

```diff
@@ -0,0 +1,60 @@
+import Gtk from "gi://Gtk";
+import Adw from "gi://Adw";
+import Gdk from "gi://Gdk";
+import GObject from "gi://GObject";
+
+const list = workbench.builder.get_object("list");
+
+// Calculate the length of our ListBox
+let last_row = list.get_last_child();
+let list_length = last_row.get_index() + 1;
+
+// Create a Dragsource and DropTarget for each row
+for (let i = 0; i < list_length; i++) {
```

Comment:
widgets are iteratables in GJS

```suggestion
// Iterate of ListBox children
for (const row of listbox) {
```